manifests/p/PearlAbyssCorp/BlackDesertKR/21.12.13.01/PearlAbyssCorp.BlackDesertKR.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: PearlAbyssCorp.BlackDesertKR
+PackageVersion: 21.12.13.01
+PackageLocale: en-US
+Publisher: PearlAbyss Corp.
+PackageName: Black Desert

```

```suggestion
PackageName: 검은사막
```
This must match Control Panel.